Formula/mongodb@3.4.rb

```diff
@@ -16,11 +16,12 @@ class MongodbAT34 < Formula
   option "with-boost", "Compile using installed boost, not the version shipped with mongodb"
   option "with-sasl", "Compile with SASL support"
 
-  depends_on "boost" => :optional
-  depends_on "go" => :build
   depends_on :macos => :mountain_lion
```

Comment:
`depends_on :macos` should come after build deps.

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/commit/0a36e614995cb4b9467c2b452c164252a9dbe7df